libscript/src/script-module.cpp

```diff
@@ -983,78 +1014,115 @@ void MCScriptReleaseRawModule(MCScriptModule *p_module)
 
 ////////////////////////////////////////////////////////////////////////////////
 
-bool MCScriptLookupPropertyDefinitionInModule(MCScriptModuleRef self, MCNameRef p_property, MCScriptPropertyDefinition*& r_definition)
+static bool
+__MCScriptLookupDefinitionInModule(MCScriptModuleRef self,
+								   MCScriptDefinitionKind p_kind,
+								   MCNameRef p_name,
+								   MCScriptDefinition*& r_definition)
```

Comment:
To simplify the callers further, could you move the following call that appears in all callers into this function?

```C++
 __MCScriptValidateObjectAndKind__(self, kMCScriptObjectKindModule);
```